cmd/az-analyze/cmd/common_test.go

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
We should rename this file too test_util.go or something like that, common_test would imply that this file contains unit tests for the functions in common.go.